src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -442,28 +442,25 @@ EscrowCreate::doApply()
         }
     }
 
-    auto const account = ctx_.tx[sfAccount];
-    auto const sle = ctx_.view().peek(keylet::account(account));
+    auto const sle = ctx_.view().peek(keylet::account(account_));
     if (!sle)
         return tefINTERNAL;
 
     // Check reserve and funds availability
     STAmount const amount{ctx_.tx[sfAmount]};
 
-    auto const balance = STAmount((*sle)[sfBalance]).xrp();
     auto const reserve =
         ctx_.view().fees().accountReserve((*sle)[sfOwnerCount] + 1);
     AccountID const issuer = amount.getIssuer();
 
-    if (balance < reserve)
+    if (mSourceBalance < reserve)
```

Comment:
`mSourceBalance` differs from `mPriorBalance` by Fee. Direct replacement for  `*sle)[sfBalance]).xrp() ` will be  `mPriorBalance` . Are you sure `mSourceBalance` is good here?

Reply:
Actually that is not true. Accessing the sle in the transactor will result in the sequence and the fee already being reduced.

```
mSourceBalance: 9999998490
mPriorBalance: 9999999990
(*sle)[sfBalance]).xrp()): 9999998490
```